website/print/docs/racer/car-label-AWE/document.inc

```diff
@@ -0,0 +1,45 @@
+<?php
+// Print 1/2 circular labels, compatible with GarageSalePup AWE.
+class CarLabelAWE extends PrintableRacerDocument {
+  function name() { return "Car Label GarageSalePup AWE"; }
```

Comment:
Could we say something like "Car number 1/2-inch round (GarageSalePup.com)" here instead?

Reply:
I changed this to "Car Number Label (GarageSalePup 1/2 inch round)"